makeSlackDark.bat

```diff
@@ -0,0 +1,3 @@
+echo off
+type static-css-injections.js >> %LOCALAPPDATA%\slack\app-3.1.1\resources\app.asar.unpacked\src\static\index.js
```

Comment:
Doesn't the `app-3.1.1` part change whenever slack updates?

Reply:
Yep. But since the theme usually needs an update anyway, there should be no problem updating all install scripts.

Reply:
Neat! Nice contrib